x-pack/plugins/alerting/server/saved_objects/maintenance_window_mapping.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsTypeMappingDefinition } from '@kbn/core/server';
+import { rRuleMappingsField } from './rrule_mappings_field';
+
+export const maintenanceWindowMappings: SavedObjectsTypeMappingDefinition = {
```

Comment:
Let's go over together but I do not think that we need add this attributes to be indexed, only the one we are going to have sort/filter/search on it. Since we are going to do all this feature in memory of the UI, I do not think that we need anything. what do you think?

Reply:
Yea sounds good to me, I'm not sure how indexing works behind the scenes, but if we need to index it again, can we add it back to the mapping?

Reply:
I will remove the explicit mappings for all fields except for `events` and `enabled` since we filter on those.